code/game/objects/items/weapons/implants/implant_loyality.dm

```diff
@@ -11,7 +11,7 @@
 				<b>Important Notes:</b> Personnel injected with this device tend to be much more loyal to the company.<BR>
 				<HR>
 				<b>Implant Details:</b><BR>
-				<b>Function:</b> Contains a small pod of nanobots that manipulate the host's mental functions.<BR>
+				<b>Function:</b> Contains a small pod of nanobots that will protect the hosts mental functions from outside influences.<BR>
```

Comment:
Alright yeah you're right. Now add to my other line comments to force Kor to fix his shizz through obnoxious peer pressure!